Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/docrepository/adapter/AdapterComponentDocRepositoryHelper.java

```diff
@@ -149,40 +148,48 @@ String extractClassificationMetadata(
+
         String classificationValue = null;
 
-        LOG.debug("Looking for classificationScheme=" + classificationSchemeUUID);
-        LOG.debug("Looking for classificationValueName=" + classificationValueName);
+        LOG.debug("Looking for classificationScheme={}", classificationSchemeUUID);
+        LOG.debug("Looking for classificationValueName={}", classificationValueName);
         // loop through the classifications looking for the desired classification uuid
-        for (oasis.names.tc.ebxml_regrep.xsd.rim._3.ClassificationType classification : classifications) {
+        for (ClassificationType classification : classifications) {
             String classificationSchemeName = classification.getClassificationScheme();
-            LOG.debug("Found classificationScheme=" + classificationSchemeName);
+            LOG.debug("Found classificationScheme={}", classificationSchemeName);
 
             if (classificationSchemeUUID.equals(classificationSchemeName)) {
-                if (classificationValueName.equals(DocRepoConstants.XDS_NAME)) {
-                    classificationValue = classification.getName().getLocalizedString().get(0).getValue();
-                } else if (classificationValueName.equals(DocRepoConstants.XDS_NODE_REPRESENTATION)) {
-                    classificationValue = classification.getNodeRepresentation();
-                } else if (classificationValueName.equals(DocRepoConstants.XDS_CLASSIFIED_OBJECT)) {
-                    classificationValue = classification.getClassifiedObject();
-                } else if (classificationValueName.equals(DocRepoConstants.XDS_CLASSIFICATION_ID)) {
-                    classificationValue = classification.getClassifiedObject();
+                switch (classificationValueName) {
+                    case DocRepoConstants.XDS_NAME:
+                        classificationValue = classification.getName().getLocalizedString().get(0).getValue();
+                        break;
+                    case DocRepoConstants.XDS_NODE_REPRESENTATION:
+                        classificationValue = classification.getNodeRepresentation();
+                        break;
+                    case DocRepoConstants.XDS_CLASSIFIED_OBJECT:
+                        classificationValue = classification.getClassifiedObject();
+                        break;
+                    case DocRepoConstants.XDS_CLASSIFICATION_ID:
+                        classificationValue = classification.getClassifiedObject();
+                        break;
+                    default:
+                        break;
                 }
-                break; // found desired classification, have values, exit loop
-            } // if (classificationSchemeUUID.equals(classificationSchemeName))
+                // found desired classification, have values, exit loop
+                break;
+            }
         }
         if (classificationValue != null && !classificationValue.isEmpty()) {
```

Comment:
Optional: since we already have NullChecker class, should we utilize to use some method(isNotNullishIgnoreSpace) in there? or using StringUtil from apache package?